examples/third_party/openssl/BUILD.openssl.bazel

```diff
@@ -66,8 +66,8 @@ configure_make(
             "libssl.lib",
         ],
         "//conditions:default": [
```

Comment:
So was this the fix? To swap the ordering here?

Reply:
Yep, I mentioned this in the commit message

Reply:
Ah, I should have looked at that... Can you also add a comment saying the ordering there is important?

Reply:
Sure no probs